omod/src/main/webapp/fragments/patientdashboard/visits.gsp

```diff
@@ -76,11 +76,25 @@
         <li class="menu-item viewVisitDetails" data-visit-id="${wrapper.visit.visitId}">
             <span class="menu-date">
                 <i class="icon-time"></i>
-                ${ui.format(wrapper.startDate)}
+                <span class="visit-start-date">
+                    <% if(ui.convertTimezones()) { %>
```

Comment:
Could you remind me as to why we are formatting a different field when convertTimezones = true?

Reply:
In this case its related to using `startDate ` or `startDatetime` when we use TZ we need the datetime information. if we dont use TZ, we only use date information ( this is the current behavior, only date)